WhatsNew/WhatsNewViewController.swift

```diff
@@ -33,6 +33,12 @@ public class WhatsNewViewController: UIViewController {
             titleLabel?.font = titleFont
         }
     }
+    /// Tint color of the feature item image, if one exists.
+    public var itemImageTintColor: UIColor = .black {
```

Comment:
Does it makes sense to have this property be optional without a default value?

I.e: `public var itemImageTintColor: UIColor? = nil`

Reply:
Now that I think about it it, it does. I'll make the adjustment.